liquidator/liquidator.js

```diff
@@ -158,12 +168,27 @@ class Liquidator {
       // Note: query the time again during each iteration to ensure the deadline is set reasonably.
       const currentBlockTime = this.empClient.getLastUpdateTime();
 
+      let tokensToLiquidate;
```

Comment:
I know this would be rare, but what happens if there were multiple liquidations in a single iteration, both of which go over the tokenBalanceWei? The answer could be that the second transaction would fail during simulation and that's acceptable since it wouldn't be able to liquidate the position anyway. Just want to get your thoughts on that case.

Reply:
Yeah I'm pretty sure the second would silently fail, just like it does currently. I'm going to simplify the test cases and make the different scenarios more clear